src/remote/run_jobs.jl

```diff
@@ -0,0 +1,198 @@
+        "num_repititions"=>repetitions
+    )
+
+    for gate in get_gates(circuit)
+        push!(
+            circuit_description["circuit"]["operations"],
+            Dict{String,Any}(
+                "type"=> get_instruction_symbol(gate),
+                "qubits"=> get_connected_qubits(gate),
+                "parameters"=> get_parameters(gate)  
+            )
+        )
+    end
+
+    circuit_json=JSON.json(circuit_description,indentation)
+
+    write(
+        joinpath(commonpath,"test_circuit.json"), 
+        circuit_json
+    )
+
+    return circuit_json
+end
+
+
+struct Client 
+    host::String
+    user::String
+    access_token::String
+end
+
+get_host(client::Client)= client.host
+
+# submits circuit to host and returns circuitID
+function submit_circuit(client::Client,circuit_json::String;verbose=false) 
+
+    path_url=joinpath(get_host(client),"circuits")
+    
+    response=HTTP.post(
+        path_url, 
+        headers=Dict(
+            "Authorization"=>"Bearer $(client.access_token)",
+            "Content-Type"=>"application/json"
+            ),
+        body=circuit_json,
+    )
+    
+    formatted_response=Dict(response)
+
+    body=JSON.parse(formatted_response["body"])
+    
+    if verbose
+        printout_response(:submit_circuit,formatted_response,typeof(response))
+    end
+
+    return body["circuitID"]
+end
+
+function printout_response(fname::Symbol,formatted_response::Dict{String,Any},type_response::Type)
+    println("\n#############################################")
+    println("\n\t$fname() returns: \n")
+
+    println("Response type: $type_response")
+
+    for (key,val) in formatted_response
+        println("")
+        println("$key :\t$val")
+    end
+
+    println("")
+end
+
+
+function get_status(client::Client,circuitID::String;verbose=false)
+
+    path_url=joinpath(get_host(client),"circuits/$circuitID")
+    
+    response=HTTP.get(
+        path_url, 
+        headers=Dict(
+            "Authorization"=>"Bearer $(client.access_token)",
+            "Content-Type"=>"application/json"
+            )
+        )
+
+    formatted_response=Dict(response)
+
+    body=JSON.parse(formatted_response["body"])
+    
+    if verbose
+        printout_response(:get_status,formatted_response,typeof(response))
+    end
+
+    return body["status"]["type"]
+end
+
+function get_result(client::Client,circuitID::String;verbose=false)
+
+    path_url=joinpath(get_host(client),"circuits/$circuitID/result")
+    
+    response=HTTP.get(
+        path_url, 
+        headers=Dict(
+            "Authorization"=>"Bearer $(client.access_token)",
+            "Content-Type"=>"application/json"
+            )
+        )
+
+    formatted_response=Dict(response)
+
+    body=JSON.parse(formatted_response["body"])
+    
+    if verbose
+        printout_response(:get_result,formatted_response,typeof(response))
+    end
+
+    return body["histogram"]
+end
+
+
+abstract type Service end
+
+struct QPUService <: Service
```

Comment:
Please rename to `AnyonQPU`

It should also only depend on client. It is going to replace the old `QPU`.